package.json

```diff
@@ -110,89 +209,142 @@
           "type": "string",
           "default": "",
           "description": "Full path to the linting rules file to use. See https://github.com/projectatomic/dockerfile_lint#extending-and-customizing-rule-files) for sample files and how to create your own."
+        },
+        "docker.showExplorer": {
+          "type": "boolean",
+          "default": true,
+          "description": "Show or hide the image and container Explorer."
+        },
+        "docker.explorerRefreshInterval": {
+          "type": "number",
+          "default": 1000,
+          "description": "Docker explorer refresh interval (default 1000ms)"
         }
       }
     },
     "commands": [
       {
         "command": "vscode-docker.configure",
-        "title": "Docker: Add docker files to workspace",
-        "description": "Add Dockerfile, docker-compose.yml files"
+        "title": "Add docker files to workspace",
+        "description": "Add Dockerfile, docker-compose.yml files",
+        "category": "Docker"
```

Comment:
Yes, it prepends the "Docker" category in the Command Palette. The reason i move do this was so that in the explorer, when you right click on a node, you don't see "Docker:" on every command.